### ISSUE TITLE
App Logic and State Management Challenge Solution

### ISSUE THREAD

**Issue by bcleland** [wix/react-native-crash-course#11]
Hi,

At the end of App Logic and State Management section, a [Challenge](https://github.com/wix/react-native-crash-course/blob/master/docs/App.Remx.md#feel-like-youre-up-for-a-challenge) is posed. I've attempted a solution to the challenge, but I'm curious to see the recommended solution. Is it possible to get the solution posted to the repo?

Thank you for creating this Crash Course and any additional information you can provide!

**Comment by drorbiran** [comment]
Hi @bcleland,

I have added the solution for the challenge:
https://github.com/wix/react-native-crash-course/commit/c026ee43ecec0a5a3db0cf57d2872e7c726fd1e7

My approach was to connect the `ViewPost` screen to the store instead of passing the post as a prop. When a screen is connected to the store, every relevant update on the store will be reflected on the screen. I also used the `AddPost` screen to edit the post.
